core/services/chainlink/config_test.go

```diff
@@ -1510,14 +1521,17 @@ func TestSecrets_Validate(t *testing.T) {
 		toml string
 		exp  string
 	}{
-		{name: "partial",
+		{
+			name: "partial",
 			toml: `
 Database.AllowSimplePasswords = true`,
 			exp: `invalid secrets: 2 errors:
 	- Database.URL: empty: must be provided and non-empty
-	- Password.Keystore: empty: must be provided and non-empty`},
```

Comment:
Not sure if there is a specific style for these, but for such cases of string literals, I always found it easier to have the curly bracket along with the string because it avoids confusion as to whether a new line character exists at the end or not.

Reply:
Yep, removed the formatting changes.